security.php

```diff
@@ -54,56 +55,148 @@ function wpcom_vip_is_restricted_username( $username ) {
 		|| WPCOM_VIP_MACHINE_USER_EMAIL === $username;
 }
 
+function _wpcom_vip_login_cache_keys( $raw_username ) {
```

Comment:
> Is there a way to make the function unavailable in php outside this file?

No - it would have to be a non-public method in a class for it to be inaccessible / privately scoped.